packages/less/test/index.js

```diff
@@ -88,3 +89,20 @@ lessTester.testSyncronous({syncImport: true}, 'math/strict/css');
 lessTester.testNoOptions();
 lessTester.testJSImport();
 lessTester.finished();
+
+(() => {
+    // Create new tester, since tests are not independent and tests
+    // above modify tester in a way that breaks remote imports.
+    lessTester = lessTest();
```

Comment:
If remote imports are being broken between tests, where do you think would be the best place to fix this? Is this a problem in Less itself? Or a problem in the way tests are set up?

Reply:
I think the problem is in the way tests are set up. Ideally tests should be independent, meaning each test should have it's own instance of `less`, even more so since the `less` instance is modified by some tests. Or, alternatively, each test should have a tear down part, where it returns `less` instance to it's initial (before test) state. However, if I understand correctly tests are also run concurrently, which might be a problem for the set up / tear down approach.

I suspect the reason remote imports get broken is because of plugin tests, might be wrong here, haven't actually investigated it.

One way to approach this is to define a function with contents of `less-node/index.js` and call it in `less-node/index.js` and before each test.

Reply:
I don't believe tests are run concurrently. They're run in a queue. But definitely it's possible they mutate the less object or don't return it to previous state. However, I doubt that's intentional, so it would be great to track it down, but maybe that's out of the scope of this issue.